test/apps/iodemo/ucx_wrapper.h

```diff
@@ -73,6 +91,15 @@ class UcxLog {
  * Holds UCX global context and worker
  */
 class UcxContext {
+    friend class UcxAcceptCallback;
+
+    class UcxAcceptCallback : public UcxEstablishCallback
```

Comment:
There are 2 reasons:
1) current design implements all the server related functionality inside `UcxContext` and it's invisible to the app.
2) as a consequence - accessibility. Callback has to modify state of the owner object, currently it's allowed by friend class definition, alternative way could be to implement another "public" API in UCX context which is relevant only for server side.

Since we have plans to refactor io demo with separating server things to listener object - lets do it there.